content/v2/tlds.markdown

```diff
@@ -47,6 +47,16 @@ List all TLDs.
 <%= pretty_print_fixture("/listTlds/success.http") %>
 ~~~
 
+### Sorting
+
+For general information about sorting, please refer to the [main guide](/v2/#sorting).
+
+Name | Description
+-----|------------
+`tld` | Sort TLDs by tld
```

Comment:
@weppos @jacegu We can actually sort TLDs by `tld`.

I understand the _Sort Foos by foo_ thing, but in the payload we return a `tld` attribute: https://github.com/aetrion/dnsimple/blob/master/spec/api/v2/serializers/suffix_serializer_spec.rb#L10